MFVNeutralino/plugins/SeedTracksProducer.cc

```diff
@@ -0,0 +1,144 @@
+    event.getByToken(museed_tracks_token, muon_seed_tracks);
+
+    edm::Handle<std::vector<reco::TrackRef>> electron_seed_tracks;
+    event.getByToken(eleseed_tracks_token, electron_seed_tracks);
+
+    //seed tracks is the main collection with all seed tracks included 
+    // neleseed and nmuseed are : of the seed tracks, how many are electrons? how many are muons? 
+    seedtks->nseedtracks = (seed_tracks->size());
+    seedtks->neleseed = (electron_seed_tracks->size());
+    seedtks->nmuseed = (muon_seed_tracks->size());
+
+    for (size_t i = 0, it = seed_tracks->size(); i < it; ++i){
+
+        const reco::TrackRef& tk = (*seed_tracks)[i];
+
+        seedtks->id.push_back(float(tk.id().id()));
+        seedtks->key.push_back(float(tk.key()));
+        seedtks->tk_idx.push_back(float(i));
+
+        int min_r = 2000000000;
+        for (int i = 1; i <= 4; ++i)
+        if (tk->hitPattern().hasValidHitInPixelLayer(PixelSubdetector::PixelBarrel,i)) {
+            min_r = i;
+            break;
+        }
+
+        double nsigmadxybs = (tk->dxy(*beamspot)) / (tk->dxyError());
+        double nsigmadz = (tk->dz(beamspot->position())) / (tk->dzError());
+
+        seedtks->p.push_back(tk->p());
+        seedtks->pt.push_back(tk->pt());
+        seedtks->pterr.push_back(tk->ptError());
+        seedtks->eta.push_back(tk->eta());
+        seedtks->etaerr.push_back(tk->etaError());
+        seedtks->phi.push_back(tk->phi());
+        seedtks->phierr.push_back(tk->phiError());
+        seedtks->minr.push_back(min_r);
+        seedtks->nhits.push_back(tk->hitPattern().numberOfValidHits());
+        seedtks->npxhits.push_back(tk->hitPattern().numberOfValidPixelHits());
+        seedtks->nsthits.push_back(tk->hitPattern().numberOfValidStripHits());
+        seedtks->npxlayers.push_back(tk->hitPattern().pixelLayersWithMeasurement());
+        seedtks->nstlayers.push_back(tk->hitPattern().stripLayersWithMeasurement());
+        seedtks->absdxybs.push_back(fabs(tk->dxy(*beamspot)));
+        seedtks->dxybs.push_back(tk->dxy(*beamspot));
+        seedtks->dxyerr.push_back(tk->dxyError());
+        seedtks->nsigmadxybs.push_back(nsigmadxybs);
+        seedtks->absnsigmadxybs.push_back(fabs(nsigmadxybs));
+        seedtks->absdz.push_back(fabs(tk->dz(beamspot->position())));
+        seedtks->dz.push_back(tk->dz(beamspot->position()));
+        seedtks->dzerr.push_back(tk->dzError());
+        seedtks->nsigmadz.push_back(nsigmadz);
+        seedtks->absnsigmadz.push_back(fabs(nsigmadz));
+        seedtks->chi2.push_back(tk->chi2());
+        seedtks->ndof.push_back(tk->ndof());
+        seedtks->chi2ndof.push_back((tk->chi2()/tk->ndof()));
+        seedtks->vx.push_back(tk->vx());
+        seedtks->vy.push_back(tk->vy());
+        seedtks->vz.push_back(tk->vz());
+        seedtks->cov.push_back(tk->covariance());
+    }
+    event.put(std::move(seedtks));
+
+}
+
+DEFINE_FWK_MODULE(MFVSeedTracksProducer);
```

Comment:
@abbywarden Is this specifically for lepton seed tracks, or is it used for all seed tracks (everywhere in the analysis) at this point? Just curious!